kotlin-react-table/src/main/kotlin/react/table/Columns.kt

```diff
@@ -51,7 +51,7 @@ class ColumnBuilder<D : Any> {
         id: IdType<D>,
         header: String,
         accessor: D.() -> V,
-        cell: (CellProps<D, V>) -> ReactElement,
+        cell: (CellProps<D, V>) -> ReactElement<*>,
```

Comment:
Are props really unknown here? Additional iterations are needed to determine concrete props?

Reply:
Yes